packages/react/src/AppShell/Header.tsx

```diff
@@ -75,7 +76,9 @@ export function Header(props: HeaderProps): JSX.Element {
           <UnstyledButton className={classes.logoButton} onClick={props.navbarToggle}>
             {props.logo}
           </UnstyledButton>
-          <HeaderSearchInput pathname={props.pathname} searchParams={props.searchParams} />
+          {!props.searchDisabled ? (
```

Comment:
I believe you can use `&&` rather than ternary here.

```tsx
{!props.searchDisabled && (
  <HeaderSearchInput pathname={props.pathname} searchParams={props.searchParams} />
)}
```